driver-core/src/main/com/mongodb/client/model/FindOptions.java

```diff
@@ -608,6 +610,32 @@ public FindOptions snapshot(final boolean snapshot) {
         return this;
     }
 
+    /**
+     * Returns the exhaust.
+     *
+     * Determines whether the returned cursor from operation execution will be an exhaust cursor.
```

Comment:
Add: If executing on a server that doesn't support exhaust cursors, this property is ignored.

Reply:
Done